source/apiVolontaria/volunteer/models.py

```diff
@@ -76,3 +86,84 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return '{}, {}'.format(self.name, str(self.address))
+
+
+class Event(models.Model):
+    """
+
+    This class represents the Event model.
+
+    """
+    class Meta:
+        verbose_name_plural = 'Events'
+
+    start_date = models.DateTimeField(
+        verbose_name="Begin date",
+        blank=False,
+        null=False,
+    )
+
+    end_date = models.DateTimeField(
+        verbose_name="End date",
+        blank=False,
+        null=False,
+    )
```

Comment:
Should we prevent `start_date` and `end_date` exceeding the dates of the `Cycle` ?

Ex:
Cycle - 12 dec. to 12 jan.
Event - 10 dec. to 20 jan.

Reply:
Good point!